.changelog/README.md

```diff
@@ -0,0 +1,132 @@
+# How To Use
```

Comment:
Should this be in the contributing guide rather than being buried in a potentially-hidden folder?

Reply:
I was conflicted about this and opted not to add it there. I can add a link from `contributing/README.md` to this document if that helps?

I think the details belong to a separate file from `contributing/README.md`, as it is missing much of the ceremony and conventions around PR management (e.g. changelog, labels, branch names).

I'm uncertain if the file is more discoverable here or in `contributing` directory. Though, I think it's more noticeable if someone navigates to this directory unexpectedly - seeing a long list of numbered txt files without README is jarring a bit.

Reply:
Ok, let's add a link from `contributing/README.md` then.